web/models.py

```
@@ -2787,3 +2787,34 @@ class ScheduledPost(models.Model):
+
+class VoiceChatRoom(models.Model):
+    """Model for voice chat rooms where users can communicate via encrypted audio."""
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=100)
+    created_by = models.ForeignKey(User, on_delete=models.CASCADE, related_name="created_voice_rooms")
+    created_at = models.DateTimeField(auto_now_add=True)
+    participants = models.ManyToManyField(User, related_name="voice_chat_rooms", blank=True)
+    is_active = models.BooleanField(default=True)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)